feature-libs/commerce-quotes/assets/translations/en/commerce-quotes.i18n.ts

```diff
@@ -60,6 +60,16 @@ export const commerceQuotes = {
       newCart: 'New Cart',
       quotes: 'Quotes',
     },
+    confirmSubmitDialog: {
+      title: 'Confirm Send Quote 000000123?',
```

Comment:
Should the quote be hard coded?

Should this be i18n alised?

Reply:
Resolved now. Thanks